packages/next/src/server/node-web-streams-helper.ts

```diff
@@ -332,13 +332,11 @@ let nextInjectToStreamBuffer: string[] = []
 export function customInjectStream(): TransformStream<Uint8Array, Uint8Array> {
   return new TransformStream({
     async transform(chunk, controller) {
+      controller.enqueue(chunk)
```

Comment:
nice, this guarantees our chunk comes before the rendered component chunks?

Reply:
I think they are sent after but are sent in the same HTML batch. Not 100% sure but seem good enough for our purposes.